### PR TITLE
Merge gitpitch presentation

### PR DESCRIPTION
**Pull request by mtorrisi**
Hello @brucellino 

what do you think about that?
Can you do a little review?

Thanks.

### CODE REVIEW

**Comment by brucellino** [comment]
sure, @mtorrisi no problem. let me have a look...

File: PITCHME.md [hunk 1]
```diff
@@ -1,25 +1,30 @@
-WRF for WIMEA-ICT
+WRF Portlet
 ===
 
 ![](assets/wrf-logo.png)
 
 ## How to use it
```
Comment (brucellino) [inline PITCHME.md]: Shouldn't this be "contributors" ?

File: PITCHME.md [hunk 2]
```diff
@@ -29,14 +34,17 @@ An example of *WPS* file of Italy.
...
 
 ---
 
-# Demo
+## Demo
 
-![Video](https://www.youtube.com/embed/vSI9fEwghHw)
+![Video](https://www.youtube.com/embed/vFFi4Yei4Eo)
 
 ---
```
Comment (brucellino) [inline PITCHME.md]: What about having a section on "what to do if you get stuck"? Add a link to the forum to discuss the page. You can even embed the comments on the page from discourse...

**Review by brucellino — COMMENTED** [review]
Some minor changes would be necessary. I would suggest having a lot of additions. where is the presentation preview  ?

File: PITCHME.md (hunk 1 above)
Comment (mtorrisi) [reply PITCHME.md]: Presentation preview is here: [url](https://gitpitch.com/sci-gaia/wimea-ict-wrf-portlet/feature)

**Comment by mtorrisi** [comment]
@brucellino I did the changes you suggest.
The presentation preview is [here](https://gitpitch.com/sci-gaia/wimea-ict-wrf-portlet/feature).

Cheers.

**Pull request merged**